accesshandler/pkg/providers/flask/access.go

```diff
@@ -0,0 +1,361 @@
+package flask
+
```

Comment:
Cleanup to be done in this file I think. A lot of very similar code used here and in the eks provider